dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-ewechat/src/main/java/org/apache/dolphinscheduler/plugin/alert/ewechat/exception/EWeChatAlertException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.alert.ewechat.exception;
+
+public class EWeChatAlertException extends RuntimeException {

```

Comment:
Consider just use `RuntimeException`, providing a separate exception makes no sense in this plugin as nobody will catch this specific exception and do recover

Reply:
Thx.